src/Components/AddTransportInfo/AddTransportInfo.js

```diff
@@ -77,17 +162,30 @@ export default class AddTransportInfo extends Component {
         <ScrollView>
           <View>
             <Text style={styles.title}>Add Transportation</Text>
-          </View>
+            </View>
```

Comment:
@lundgrea Let's use a package for radio buttons here to make sure only valid inputs are passed to the fetch call

Reply:
Based on the update to the db we are able to add any type of transportation. I did not install a package.